cmd/opm/index/add.go

```diff
@@ -96,15 +98,29 @@ func runIndexAddCmdFunc(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	containerTool, err := cmd.Flags().GetString("container-tool")
+	buildTool, err := cmd.Flags().GetString("build-tool")
 	if err != nil {
 		return err
 	}
 
-	if containerTool == "none" {
+	if buildTool == "none" {
 		return fmt.Errorf("none is not a valid container-tool for index add")
 	}
 
+	pullTool, err := cmd.Flags().GetString("pull-tool")
+	if err != nil {
+		return err
+	}
+
+	containerTool, err := cmd.Flags().GetString("container-tool")
+	if err != nil {
+		return err
+	}
+
+	if containerTool != "" {
```

Comment:
Any thoughts on how backwards compatible we want to be? If we explode here we end up breaking existing clients. Though we can do a major version bump after to be dogmatic about semver.

Reply:
I was sort of back and forth on this. I think that's fair, maybe we want to just treat this as an alias and use it to overwrite defaults if the two new flags haven't been specified?

Reply:
SGTM, but is there a convention for telling the difference between unset and set-to-default?